templates/bot/ts/default/.env.teamsfx.local

```diff
@@ -0,0 +1,17 @@
+# TeamsFx will overwrite the following variable values when running debug
+BOT_ID=
+BOT_PASSWORD=
+M365_CLIENT_ID=
+M365_CLIENT_SECRET=
+M365_TENANT_ID=
+M365_AUTHORITY_HOST=
+INITIATE_LOGIN_ENDPOINT=
+API_ENDPOINT=
+M365_APPLICATION_ID_URI=
+
+# Following variables can be customized or you can add your owns
+# FOO=BAR
+SQL_ENDPOINT=
```

Comment:
can we remove SQL related items in this file?

Reply:
Updated